### DISCUSSION TITLE
The html solutions that don't render correctly should not be valid.

### DISCUSSION THREAD

**Discussion by CooleyGit** [cssbattle/forum#57] [General]
I was wondering how people were getting such low char count then I found example solutions that seem to be HTML that doesn't render correctly? Should this not be a prerequisite that the winning solution actually renders in normal browsers? Kinda lame and not really real-world CSS... I mean it's an interesting solution but not really something a human would write. Would almost be better if u define the HTML (one div) and all you can do is write CSS and your score is based on ur CSS. 
<img width="774" alt="Screen Shot 2021-05-17 at 3 34 21 PM" src="https://user-images.githubusercontent.com/3848152/118553660-36258180-b726-11eb-9971-8473a5f11128.png">

**Comment by mielipuolinen** [comment]
I agree. Funnily enough I came here because of this exactly same target. How would I even design something like that if I don't know how the server side will render it and if that's only thing that matters in scoring? In this case this wouldn't render to anything similar either on FF, Chrome or Edge. 

I'd like to propose to have another category for more credible solutions. Maybe with a requirement that a solution should work on all major browsers and why not even a requirement to follow some standards? I guess in this kind of category more advanced targets would be needed for more diversity between solutions. However I feel like this is what should matter in CSS competitions and could have some real world applications as well.

But of course, in these kinds of competitions it always ends up to abusing something and stopping that would be just a cat-and-mouse game.

Otherwise I've enjoyed really much doing these tasks. Awesome puzzles!

**Comment by bmerigan** [comment]
Yes. I upvote this.  The winning "solutions" are ridiculous hacks which do not represent real CSS.  I'm dubious whether they are even written by a human.
